### DISCUSSION TITLE
Sending heart rate to Zwift broadcasted from Garmin watch

### DISCUSSION THREAD

**Discussion by toniwong** [cagnulein/qdomyos-zwift#840] [Q&A]
Hi All,

Can gdomyos-zwift receive the heart rate broadcasted from my Garmin watch and then transmits to Zwift? 

Have tested the USB ANT+ stick which successfully transmits the HR to Zwift with other ANT+toBLE APK with my phone. However the APK can't run simultaneously with gdomyos-zwift. Now I have to use 2 phones for HR &  gdomyos-zwift separately.

Treadmill: Kingsmith WalkingPad R1 Pro
Gdomyos-zwift: LG V40 Android phone with USB ANT+ stick
Zwift: iPad Air 5
Watch: Garmin 645 Music (ANT+ only)

Many thanks~!

**Comment by cagnulein** [comment]
yes qz can receive ant signal from the garmin on the android phones with ant. just enable the ant heart setting. let me know

**Reply by toniwong** [reply]
Both my phones are not built-in with ANT+. A USB ANT+stick is connected to the phone for receiving the heart rate from the Garmin watch. 

Can QZ receive the HR via the connected USB ATN+ stick and broadcast to Zwift, together with the treadmill speed?

**Reply by cagnulein** [reply]
nope unfortunately. Not yet. There is an open ticket about this https://github.com/cagnulein/qdomyos-zwift/issues/552

**Reply by toniwong** [reply]
Thanks~!
